app.config.yaml

```diff
@@ -35,3 +35,13 @@ application:
               IMS_PRODUCT_CONTEXT: $IMS_PRODUCT_CONTEXT
             limits:
               timeout: 180000
+          target:
+            function: actions/target/index.js
+            web: true
+            runtime: nodejs:18
+            inputs:
+              TARGET_API_KEY: $TARGET_API_KEY
```

Comment:
I didn't see the value of this environment variable being to the GitHub workflow actions for QA and Prod.

Reply:
Technically, the key was secretly generated specifically for our AIO project, but I do not know if there is a security policy that defines the rules and about any potential attack vectors.
BTW We also do the same for the FIREFALL_API_KEY.
Putting this key in the code would mean making it public on GitHub.

Reply:
> Putting this key in the code would mean making it public on GitHub.

This means that anyone with this key and valid access token can pretend to be the Generate Variations app from the Target POV.

Reply:
Another reason to put the key in GitHub is that it is generated per environment.

Reply:
Okay. I was just wondering if you forgot to add it to Github. If it will injected by Github as environment variable then, this makes sense.